src/components/VmDetails/cards/UtilizationCard/index.js

```diff
@@ -45,8 +45,8 @@ const UtilizationCard = ({ vm }) => {
             }
           </Col>
           <Col>
-            { vm.has('disks')
-              ? <DiskCharts vm={vm} isRunning={isRunning} id={`${idPrefix}-disk`} />
+            { vm.has('disks') && stats.disks
```

Comment:
If `stats.disks` is there, you won't be able to show the allocated/provisioned details for VMs without a guest agent.

Plus the checks in this component are really to guard against having the base VM data only and waiting for the VM's deep fetch to complete.